### PR TITLE
CB-10294: Wrong link in docs

### PR DESCRIPTION
**Pull request by sarangan12**
Fixed one wrong link in docs and also changed the links to reference links.

@dblotsky @riknoll @nikhilkh @rakatyal Can you please review and merge this PR?

### CODE REVIEW

File: www/docs/en/dev/cordova/storage/storage.md [hunk 1]
```diff
@@ -64,6 +62,12 @@ The following platforms support IndexedDB:
...
-[Cordova plugins](http://plugins.cordova.io/) provide similar storage options.
+[File API][FileAPI] allows you to cache data on the local file system.  Other
+[Cordova plugins][Cordovaplugins] provide similar storage options.
+
+[Html5RocksStorage]: http://www.html5rocks.com/en/features/storage
+[W3CSpecStorage]: http://www.w3.org/TR/webstorage/
+[WebSQLDatabaseSpecification]: http://dev.w3.org/html5/webdatabase/
+[IndexedDB]: http://www.w3.org/TR/IndexedDB/
+[FileAPI]: https://github.com/apache/cordova-plugin-file/blob/master/README.md
+[Cordovaplugins]: /plugins
```
Comment (riknoll) [inline www/docs/en/dev/cordova/storage/storage.md]: This link should use `{{ site.baseurl }}`. That is,
```
[Cordovaplugins]: {{ site.baseurl }}/plugins 
```

Comment (riknoll) [inline www/docs/en/dev/cordova/storage/storage.md]: Also maybe capitalize the 'p'

**Comment by rakatyal** [comment]
Are you using 'checklinks' gulp task to find these broken links? I believe we have [this] (https://issues.apache.org/jira/browse/CB-10058) Jira to fix the broken links. 
@dblotsky, @riknoll : I believe we still have the checklinks task in our gulpfile?

**Comment by sarangan12** [comment]
@riknoll Addressed the comments

**Comment by sarangan12** [comment]
@rakatyal These broken links are found using the JIRA tickets created. At this point, I did check the 'checklinks' task. It does not work with all cases. I am planning to take a look at it soon

**Comment by rakatyal** [comment]
Okay great. Thanks. And this looks good to me.

**Pull request closed without merging**